health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -306,6 +306,28 @@ export const UICustomizations = {
+                  `/${window.contextPath}/employee/microplan/select-activity?microplanId=${row?.id}&campaignId=${row?.campaignId}`
+                );
+                const navEvent2 = new PopStateEvent("popstate");
+                window.dispatchEvent(navEvent2);
+                break;
+              case t("WBH_DOWNLOAD_MICROPLAN"):
+                handleFileDownload();
+                break;
+              default:
+                break;
+            }
+          };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)